src/gordon_gcp/plugins/janitor/reconciler.py

```diff
@@ -240,6 +240,33 @@ def _parse_rrset_message(self, message):
 
         return zone, rrsets
 
+    @staticmethod
+    def _remove_soa_and_root_ns(zone, rrsets):
+        """Given a list of GCPResourceRecordSets, remove SOA and root NS.
+
+        This is necessary because Google won't let you override its SOA and root
+        NS records.
+
+        Returns a new list (doesn't modify in place).
+
+        Args:
+            zone (str): The domain name the rrsets are for, including trailing
+                dot.
+            rrsets (list[GCPResourceRecordSet]): The GCPResourceRecordSets to
+                filter.
+
+        Returns:
+            (list[GCPResourceRecordSet]): The filtered GCPResourceRecordSets.
```

Comment:
do we need the parenthesis around this?

Reply:
Hm.  I guess not.  I seem to be remembering the style guide wrong.